apps/dot_ext/forms.py

```diff
@@ -125,6 +125,14 @@ def clean_redirect_uris(self):
                         raise forms.ValidationError(msg)
         return redirect_uris
 
+    def save(self, commit=True):
```

Comment:
don't override like this, it's unclear what's happening and save's, without commit? and then with it.

Reply:
Placed the ToS agree logging back under a save() override, since values for user and app name were not getting set yet in the clean_agree().   This works for new application registration. 

I moved the application's op_tos_uri setting to the model's field default instead of here.